src/macros.cr

```diff
@@ -185,3 +185,27 @@ macro p!(*exps)
     }
   {% end %}
 end
+
+# Raises an exception.
+# Useful to signal an unreachable section of the code.
+#
+# ```
+# values = ["a", "b"]
+# idx = Random.rand(2)
+#
+# case values[idx]
```

Comment:
This is a bit shorter and similarly expressive:
```suggestion
# case values.sample
```

Reply:
I was looking for this exact method, but I didn't found it.